### PR TITLE
Good

### PR DESCRIPTION
**Pull request by iUnstable0**
wow!

### CODE REVIEW

**Comment by jessiejs** [comment]
pefection

**Comment by stohrendorf** [comment]
Although I'm not a parent myself, I think it should be noted that baby mode prevents them from swallowing exceptions, and that it disables hazardous language constructs and sharp edge cases.

**Comment by definitely-nobody-is-here** [comment]
I don't see why this hasn't been merged yet, this is possibly the best idea to have ever been ideated. Perhaps a few more safety precautions would be useful, but this should be implemented.

File: README.md [hunk 1]
```diff
@@ -726,6 +727,133 @@ It's worth noting that Github Copilot doesn't understand DreamBerd, which means
...
+goo(x)! // 3
+```
+
+However in Baby Mode, 🍌🍌🍌 (banana-banana-banana) is not the same as 🍎🍎🍎 (apple-apple-apple), even though they both represent the number 3! Why? Because a banana is not an apple! And so, we teach babies the concept of object identity in a fun, intuitive way.
+
+```java
+goo(age == x)! // false
+```
+
+To keep things simple and fun, Baby Mode supports a limited set of DreamBerd's functions
```
Comment (stohrendorf) [inline README.md:807]: This is missing a dot at the end.

**Pull request closed without merging**